samples/benchmarks/coremark/prj_flash_and_run.conf

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+# Common coremark configuration
+############################################################################
```

Comment:
I guess there is no way to reuse common configuration :(
I don't like coping it but understand there is no better solution available if we want to use file sufix?

Reply:
You can define a promptless `APP_COREMARK` Kconfig at the sample level that selects the common configuration and overrides the defaults for non-selectable options. However, I decided that this copy-paste approach is better as it cleanly demonstrates what needs to be enabled in the .conf file for the CoreMark sample.